openai/api_resources/abstract/api_resource.py

```diff
@@ -1,11 +1,15 @@
 from urllib.parse import quote_plus
 
 from openai import api_requestor, error, util
+import openai
 from openai.openai_object import OpenAIObject
+from openai.util import ApiType
 
 
 class APIResource(OpenAIObject):
     api_prefix = ""
+    azure_api_prefix = 'openai/deployments'
```

Comment:
I'd rather that this is a prefix in code. The reason is that if we somehow change this later down the line, it is just a very small PR rather than asking all customers that are using this to change their env variable..

Reply:
That's fair. I guess I thought there was some SDK wrapping around this client from msft side, so it would just be a change there instead of requiring all users to update.